core/shared/src/main/scala/com/lynbrookrobotics/potassium/units/spacialValues.scala

```diff
@@ -0,0 +1,58 @@
+package com.lynbrookrobotics.potassium.units
+
+import squants.space.{Area, Feet, Length}
+
+class Point(override val x: Length,
+            override val y: Length,
+            override val z: Length) extends Value3D(x, y, z) {
+  def this(value3D: Value3D[Length]) = this(value3D.x, value3D.y, value3D.z)
+  def this(x: Length, y: Length) = this(x, y, Feet(0))
+  def distanceTo(other: Point): Length = Segment(this, other).length
+
+  def dot(other: Point): Area = {
+    this.x * other.x + this.y * other.y + this.z * other.z
+  }
+
+  def magnitude: Length = (x * x + y * y + z * z).squareRoot
+
+  def zip(other: Point) = Segment(this, other)
+
+  // Better solution, perhaps using type aliases?
```

Comment:
Not really (unless you want to go by the `CanBuildFrom` route). If you integrate the other methods directly into `Value3D` you won't need this though.